### PR TITLE
Add confidence lines

### PR DESCRIPTION
**Pull request by jrapin**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

So far there was no confidence information on benchmark plots. This adds confidence area in transparency to prevent overloading plots (too much...).

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/plotting.py [hunk 1]
```diff
@@ -243,6 +248,16 @@ def __init__(
...
         # self._fig.tight_layout()
 
+    @staticmethod
+    def _get_confidence_arrays(vals: Dict[str, np.ndarray], log: bool = False) -> Tuple[np.ndarray, np.ndarray]:
+        loss = vals["loss"]
+        conf = vals["loss_std"] / np.sqrt(vals["num_eval"] - 1)
+        if not log:
+            return loss - conf, loss + conf
+        lloss = np.log10(loss)
+        lstd = 0.434 * conf / loss
```
Comment (jrapin) [inline nevergrad/benchmark/plotting.py]: This should be crosschecked

Comment (jrapin) [reply nevergrad/benchmark/plotting.py]: For the record: https://faculty.washington.edu/stuve/log_error.pdf

Comment (teytaud) [reply nevergrad/benchmark/plotting.py]: Why not simply << return loss - conf, loss + conf >> so that we plot << x, log(y+-std) >> ?
The goal is to plot something inspired from << x, log(y) +- std(log(y)) >> rather than << x, log(y +- std(y)) >> ?

Comment (jrapin) [reply nevergrad/benchmark/plotting.py]: From https://faculty.washington.edu/stuve/log_error.pdf it seems it's not a good practice to use log(y+-std).
From what I have observed, it does look better as well.
